applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -107,6 +117,9 @@ static uint32_t img_length;
 
 static uint16_t store_offset;
 static uint16_t sync_offset;
+static int flash_write_block;
```

Comment:
this should be `uint32_t` -> `uint32_t flash_area_align(const struct flash_area *fa);`